src/components/Dashboard/UpcomingBookings.vue

```diff
@@ -14,7 +14,11 @@ const obsPortalDataStore = useObsPortalDataStore()
 
 const currentPage = ref(1)
 const sessionsPerPage = 5
-const requestGroups = ref([])
+
+const pendingScheduledObservations = computed(() => {
```

Comment:
Is there a reason this needs to be a computed property rather than just using the variable in the store directly?